contracts/interfaces/IPausable.sol

```diff
@@ -0,0 +1,14 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.28;
+
+/// @title IPausable
+/// @notice Interface for pausable contracts for internal use.
+interface IPausable {
```

Comment:
A couple of questions:
1. Why would this need another interface? It's only used in one contract 🤷
2. Feels like there should be a query for isPaused?